packages/web/src/components/treeView.css

```diff
@@ -69,6 +69,13 @@
   color: var(--vscode-list-activeSelectionForeground) !important;
 }
 
+.tree-view-content:focus .tree-view-entry.selected button.eye.toggled {
+  color: var(--vscode-button-secondaryForeground) !important;
+  border-radius: 5px;
```

Comment:
Where did you pick this value from? While the project is not particularly consistent, it looks like we tend to use `6px` around the UI.